Topics/Tech_Stacks/Odata.md

```diff
@@ -10,7 +10,7 @@
 
 ### [OData Basics](#odata-basics-1)
 
-### [How to Use OData for RESTful APIs](#odata-basics-1)
+### [How to Use OData for RESTful APIs](#how-to-use-odata-for-restful-apis-1)
```

Comment:
This table of contents file doesn't link correctly.

Reply:
Good catch. I will fix that. Thanks Max!